CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ Currently, this repo is in Prerelease. When it is released, this project will ad
 ### Header V2
 
 - Adds a third radio button, separating search options between the Circulating Catalog, Research Catalog, and the NYPL.org website.
+- Temporarily removes the "log in" state, the feature to read auth cookies, and the patron's name when logged in, and the log out link.
```

Comment:
Why is this temporary?

Reply:
Vega does not support NYPL cookies at the moment so the Header, when embedded in Vega, cannot read the auth cookie and show/display that the patron is logged in. So the decision was made to remove this feature until Vega can support it. That's why, for now, the button will read "My Account" instead of "Log In", the patron's name won't be displayed when a patron is signed in, and there will be no "Log Out" button.

Reply:
OK, thanks for the explanation.  You may have explained this in the past, but I don't think I fully understood.